packages/neon-core/src/u/StringStream.ts

```diff
@@ -83,6 +83,10 @@ export class StringStream {
     return len;
   }
 
+  public readUint32(): number {

```

Comment:
let's not extend the public interface of StringStream in this PR. We can open another PR for this if there are multiple use cases within the repository.